App/Sagas/ThreadsSagas.ts

```diff
@@ -50,9 +50,9 @@ export function * acceptExternalInvite (action: ActionType<typeof ThreadsActions
   }
 }
 
-export async function getDefaultThread (): Promise<Thread | undefined> {
+export async function getDefaultThread (): Promise<Thread> {
   const threads = await TextileNode.threads()
-  return threads.items.find(thread => thread.name === 'default')
+  return threads.items.find(thread => thread.name === 'default')!
```

Comment:
We just assume the thread exists since we know we created it when the node was started, but before any sagas that use the default thread run